anvio/docs/artifacts/anvi-setup-cazymes.md

```diff
@@ -0,0 +1,17 @@
+This program **downloads and organizes a local copy of the dbCAN [CAZyme HMMs database](https://bcb.unl.edu/dbCAN2/download/Databases/) for function annotation.** This program generates a %(cazymes-data)s artifact, which is required to run the program %(anvi-run-cazymes)s. 
+
+### Set up CAZyme data
+{{ codestart }}
+anvi-setup-cazymes 
+{{ codestop }}
+
+By default, this data is stored at `anvio/data/misc/CAZyme/`. To set up this data in a non-default location, run 
+{{ codestart }}
+anvi-setup-cazymes --cazyme-data-dir path/to/location
+{{ codestop }}
+
```

Comment:
I suggest adding a note in this section that if people want to download multiple versions of the CAZyme database, they can use different `--cazyme-data-dir`s to do it